app.rb

```diff
@@ -0,0 +1,84 @@
+require 'sinatra'
+require 'sinatra/reloader'
+require 'http'
+require 'rack-livereload'
+require 'lazyrecord'
+require_relative 'model/Book'
+
+use Rack::LiveReload if development?
+
+@@key = "AIzaSyC8xY7H9mf9HVrAmFPcb66v9hWeW4asmBU"
+helpers do
+  def get_books(params)
+    JSON.parse(HTTP.get("https://www.googleapis.com/books/v1/volumes?q=#{params}&key=#{@@key}").body)
+  end
+
+  def get_book_no_desc(id)
+    url = "https://www.googleapis.com/books/v1/volumes/#{id}?fields=volumeInfo(title,authors,imageLinks/thumbnail)&key=#{@@key}"
+    JSON.parse(HTTP.get(url).body)
+  end
+
+  def get_specific_book(id)
+    JSON.parse(HTTP.get("https://www.googleapis.com/books/v1/volumes/#{id}?key=#{@@key}").body)
+  end
+
+  def get_params(id)
+    book_json = get_specific_book(id)
+    arr = {}
+    arr["id"] = book_json["id"]
+    arr["img"] = book_json["volumeInfo"]["imageLinks"]["thumbnail"]
+    arr["title"] = book_json["volumeInfo"]["title"]
+    arr["author"] = book_json["volumeInfo"]["authors"]
+    arr
```

Comment:
Maybe this hash could have been created in a literal instead of creating the hash to later add values to it?

```suggestion
    arr = {
      id: book_json["id"],
      img: book_json["volumeInfo"]["imageLinks"]["thumbnail"],
      title: book_json["volumeInfo"]["title"]
      author: book_json["volumeInfo"]["authors"]
    }
```